app/serializers.py

```diff
@@ -277,8 +283,23 @@ class Meta:
             "is_superuser",
             "headline",
             "contact_number",
+            "allowed_modules",
         ]
 
+    def get_allowed_modules(self, data):
+        allowed_modules = []
```

Comment:
why not just make it set 
allowed_modules = {}
allowed_modules.add(module.slug)